DMCompiler/DM/DMObjectTree.cs

```diff
@@ -9,6 +10,9 @@ static class DMObjectTree {
         public static List<string> StringTable = new();
         public static Dictionary<string, int> StringToStringID = new();
         public static DMProc GlobalInitProc = new DMProc(null);
+        public static bool ThrowOnError = false;
+        public static bool PrintPathOnError = true;
+        public static bool PrintUnimplementedWarnings = false;
```

Comment:
I don't like the idea of behavior being controlled by hard-coded debug switches hidden somewhere in the code. Could these be removed?